upgrade/Upgrade-3.1.10.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * 2007-2014 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    EnvoiMoinsCher <informationapi@boxtale.com>
+ * @copyright 2007-2014 PrestaShop SA / 2011-2014 EnvoiMoinsCher
+ * @license   http://opensource.org/licenses/afl-3.0.php Academic Free License (AFL 3.0)
+ * International Registred Trademark & Property of PrestaShop SA
+ */
+
+function upgrade_module_3_1_10($module)
+{
+	//Set default configuration
+	Configuration::updateValue('EMC_ENABLED_LOGS', 0);
+	$states = OrderState::getOrderStates((int)$module->getContext()->language->id);
+	$states_array = array();
+	foreach ($states as $state)
+		array_push($states_array, $state['id_order_state']);

```

Comment:
As I said on the first validation of this PR, using ```$states_array[] = $state['id_order_state'];``` is better if you need to add only one value.